src/renderer/redux/selectors/app.js

```diff
@@ -127,124 +141,92 @@ export const selectNavLinks = createSelector(
 
     // Gets the last active sublink in a section
     const getActiveSublink = category => {
-      if (category === 'wallet') {
+      if (category === PAGES.WALLET) {
         const previousPath = getPreviousSubLinkPath(isWalletPage);
-        return previousPath || '/wallet';
-      } else if (category === 'myLbry') {
-        const previousPath = getPreviousSubLinkPath(isMyLbryPage);
-        return previousPath || '/downloaded';
+        return previousPath || `/${PAGES.WALLET}`;
       }
 
       return undefined;
     };
 
-    const isCurrentlyWalletPage = isWalletPage(currentPage);
-    const isCurrentlyMyLbryPage = isMyLbryPage(currentPage);
+    // Is this path the first unacknowledged item in the guided tour list
+    const getGuideIfNecessary = page => {
+      if (!firstRunCompleted) {
+        return null;
+      }
+      return guidedTourItem && guidedTourItem.page === page ? guidedTourItem.guide : null;
+    };
+
+    const buildLink = (label, page) => ({
+      label,
+      path: `/${page}`,
+      active: currentPage === page,
+      guide: getGuideIfNecessary(page),
+    });
 
     const walletSubLinks = [
       {
-        label: 'Overview',
-        path: '/wallet',
-        active: currentPage === 'wallet',
+        ...buildLink('Overview', PAGES.WALLET),
```

Comment:
Could make an array of these pairs and do a map.

Reply:
Ah doh. Good idea.

Reply:
Eh I think I'll leave it. Some of the links are slightly different, and then I would need to change all of those into objects, so there will still be some repetition.